auth/mock.go

```diff
@@ -28,13 +26,6 @@ func MockMiddleware(next echo.HandlerFunc) echo.HandlerFunc {
 
 		ctx = ctx.WithUser(&models.Person{ID: person.ID, Email: person.Email})
 
-		var agent models.Agent
-		if err := ctx.DB().Where("person_id = ?", person.ID.String()).First(&agent).Error; err != nil && !errors.Is(err, gorm.ErrRecordNotFound) {
-			return err
-		} else {
-			ctx = ctx.WithAgent(agent)
-		}
-
```

Comment:
In this mock auth middleware, removed setting the agent in the context so we explicitly test that the new agent auth middleware sets it.

The test passes even if this isn't removed.